bundles/core/org.eclipse.smarthome.core.thing.xml.test/src/test/resources/test-bundle-pool/SystemChannels.bundle/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,10 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Synthetic Test Bundle
+Bundle-SymbolicName: SystemChannels.bundle;singleton:=true
+Bundle-Vendor: Eclipse.org/SmartHome
+Bundle-Version: 0.7.0.qualifier
```

Comment:
Yes, I think it was initially done by mistake, also because a MANIFEST.MF from an original ESH bundle was copied. We can use any version there, because the tests don´t care about the version. So maybe you can also change it in the other manifest files.